tests/cypress/integration/actions_users/issue_1810_login_logout.js

```diff
@@ -71,5 +75,32 @@ context('When clicking on the Logout button, get the user session closed.', () =
                 cy.contains('.cvat-task-details-task-name', `${taskName}`).should('be.visible');
             });
         });
+
+        it('Incorrect user and correct password', () => {
+            cy.logout();
+            if (Cypress.browser.family !== 'chromium') {
+                cy.get('.cvat-modal-unsupported-platform-warning').within(() => {
+                    cy.contains('button', 'OK').click();
+                });
+            }
+            login('randomUser123', Cypress.env('password'));
+            cy.url().should('include', '/auth/login');
+            cy.get('.cvat-notification-notice-login-failed').should('exist');
+            cy.closeNotification('.cvat-notification-notice-login-failed');
```

Comment:
closeNotification already contains cy.get(className). Do we need this duplication? (the same in the code below)

Reply:
Unnecessary actions removed.